pantab/tests/conftest.py

```diff
@@ -81,15 +79,15 @@ def df():
             "int16",
             "int32",
             "int64",
-            "Int16",
-            "Int32",
-            "Int64",
+            # "Int16",
+            # "Int32",
+            # "Int64",
             "float32",
             "float64",
-            "bool",
+            # "bool",
             "datetime64",
-            "datetime64_utc",
-            "timedelta64",
+            # "datetime64_utc",
+            # "timedelta64",
```

Comment:
timedelta support will be broken in 4.0. the pandas buffer interface doesn't send it, and there is some ambiguiuty across different libraries as to how this is managed anyway